package.json

```diff
@@ -36,7 +35,7 @@
     "ts-node": "3.2.0",
     "tslint": "4.4.2",
     "tslint-config-standard": "4.0.0",
-    "typescript": "^2.7.1"
+    "typescript": "^2.8.3u"
```

Comment:
is this a typo?

Reply:
Yes, good catch. I saw that but I was sure I removed that :)